scraper/src/mindtouch2zim/constants.py

```diff
@@ -9,8 +9,12 @@
 VERSION = __version__
 ROOT_DIR = pathlib.Path(__file__).parent
 
-# Loading the CSS leads to many bad assets at these URLs, we just ignore them
-STANDARD_KNOWN_BAD_ASSETS_REGEX = r"https?:\/\/a\.mtstatic\.com/@(cache|style)"
+# Loading the CSS leads to many bad assets at a.mtstatic.com/@cache or
+# a.mtstatic.com/@style we just ignore them
+# Multiple images are badly loaded from localhost (=> no need to retry these)
+STANDARD_KNOWN_BAD_ASSETS_REGEX = (
+    r"https?:\/\/(a\.mtstatic\.com\/@(cache|style)|localhost(:|\/))"
```

Comment:
This is becoming hard to maintain. a list will eventually be necessary

Reply:
I eventually agree ^^
Until then, I've added automated cases to at least validate this more easily